OSDOCS-2024: Remove rootfs encrypt bullet (#34016)
openshift/openshift-docs: modules/installation-special-config-storage.adoc

. Save the Butane config in case you need to update the manifest in the future.

. Continue with the remainder of the {product-title} deployment.

[IMPORTANT]
====

> If you configure additional data partitions, they are only encrypted when encryption is explicitly requested.

Reviewer: Suggestion to strengthen the language a bit:

```suggestion
If you configure additional data partitions, they will not be encrypted unless encryption is explicitly requested.
```

Author: Thanks, updated to your original wording 😀

Reviewer: ...whoops, I didn't even notice. :sweat_smile:
